docs/community/content/contribute/document-contributor.cn.md

```diff
@@ -98,20 +98,20 @@ ShardingSphere 文档使用 Markdown 编写，并使用 Hugo 进行处理生成
 
 1. 附加出现问题的文档链接。
 
-1. 详细描述问题。
+2. 详细描述问题。
```

Comment:
as a matter of fact, this is not "an error", github markdown engine will render the document to what we expected even the order of number are all 1.

anyway, thanks for your contribution to "fix" this.